tracer/test/Datadog.Trace.IntegrationTests/ContainerTaggingTests.cs

```diff
@@ -51,7 +51,7 @@ public async Task Http_Headers_Contain_ContainerId()
 
                 var headers = agent.TraceRequestHeaders.Should().ContainSingle().Subject;
 
-                var expectedContainedId = ContainerMetadata.GetContainerId();
+                var expectedContainedId = ContainerMetadata.Instance.ContainerId;
```

Comment:
The idea in general is to _not_ use `ContainerMetadata.Instance` anywhere, other than at the "root" in the real application, which injects it in 🙂 Particularly in tests, you 100% don't want to be using the static instance. For example:

```suggestion
                var containerMetadata = new ContainerMetadata();
                var expectedContainedId = containerMetadata.ContainerId;
```